library-app/src/services/AuthorServices.tsx

```diff
@@ -0,0 +1,13 @@
+import { AxiosResponse } from 'axios'
+import Author from '../models/author.model'
+import axiosInstance from './axiosConfig'
+
+const getAuthors = async (): Promise<AxiosResponse> => {
+  return axiosInstance.get<Author[]>('api/Authors', {
+    headers: {
+      Authorization: `Bearer ${localStorage.getItem('accessToken')}`,
```

Comment:
You should use interceptors for authorization instead of adding headers maually each time